tools/toit.toit

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2024 Toitware ApS.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; version
+// 2.1 only.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+import cli
+import fs
+import host.file
+import host.pipe
+import system
+
+main args/List:
+  if args.size > 0 and args[0].ends-with ".toit":
+    args = ["run", "--"] + args
+
+  root-command := cli.Command "toit"
+      --help="The Toit command line tool."
+      --options=[
+        cli.Option "sdk-dir"
+            --help="Path to the SDK root."
+            --type="dir"
+            --hidden,
+      ]
+
+  compile-analyze-run-options := [
+    cli.Flag "show-package-warnings"
+        --help="Show warnings from packages.",
+    cli.Flag "Werror" --short-name="Werror"
```

Comment:
So the short name means you can write `--Werror` and `-Werror`?

Reply:
correct.